templates/pages/admin/form/access_control.html.twig

```diff
@@ -91,7 +89,7 @@
                             >
                         </label>
                     </h3>
-                    <div>
+                    <div class="access-control-config-form">
```

Comment:
```suggestion
                    <div data-glpi-access-control-form>
```

Reply:
This reuse the now useless "data-glpi-access-control-form" tag and avoid using a class which its only purpose is to be a selector.